app/views/course/assessment/assessments/show.html.slim

```diff
@@ -34,6 +34,21 @@
       tr.end_at
         th = t('.end_at')
         td = format_datetime(@assessment.end_at) if @assessment.end_at.present?
+      - if can?(:manage, @assessment)
+        - if @assessment.autograded?
+          tr.skippable
+            th = t('.skippable')
+            td = @assessment.skippable ? t('.allow') : t('.disallow')
+          tr.allow_partial_submission
+            th = t('.allow_partial_submission')
+            td = @assessment.allow_partial_submission ? t('.allow') : t('.disallow')
+        tr.graded_test_case_types
+          th = t('.graded_test_case_types')
+          - used_test_types = []
```

Comment:
Move this and the following logic into `app/helpers/course/assessment/assessments_helper.rb`

So the slim file here should just be  `td = format_inline_text(display_graded_test_case_types(@assessment))`